src/Datadog.Trace.ClrProfiler.Managed/Integrations/WebRequestIntegration.cs

```diff
@@ -115,7 +115,13 @@ public static object GetResponse(object webRequest, int opCode, int mdToken, lon
         /// <param name="moduleVersionPtr">A pointer to the module version GUID.</param>
         /// <returns>Returns the value returned by the inner method call.</returns>
         [InterceptMethod(
-            TargetAssembly = "System.Net",
+            TargetAssembly = "System", // .NET Framework
+            TargetType = WebRequestTypeName,
+            TargetSignatureTypes = new[] { "System.Threading.Tasks.Task`1<System.Net.WebResponse>" },
+            TargetMinimumVersion = Major4,
+            TargetMaximumVersion = Major4)]
+        [InterceptMethod(
+            TargetAssembly = "System.Net.Requests", // .NET Core
```

Comment:
Maybe this change was already in the branch when I handed it off, but is this change necessary for this PR or can it be moved out?

Reply:
The change from `System.Net` to `System` seems weird. Was this integration not working on .NET Framework?

Reply:
It was already in the branch, yes. Let's create a card to follow up on it?

Reply:
Oh wait, this was intended to be here!! I vaguely remember that this identified an outgoing http call made by the AWS sdk.

Reply:
Yeah I think this was not working on .NET Framework. Colin, since you're actively working on this branch, could you verify that removing this integration causes no "http" spans to show up on the .NET Framework calls? There should basically be an "aws" span wrapping an "http" span for each call

Reply:
My concern is that by changing this from `System.Net` to `System` we will no longer instrument outgoing http requests in systems where `WebRequest` lives in `System.Net.dll`. Should we add both to cover all the bases?

Reply:
I'm totally happy to have both. What do you think?

Reply:
I don't think we need `System.Net`. This seems like the right fix, thanks. (resolving)